src/contexts/MemeContext.js

```diff
@@ -0,0 +1,16 @@
+import { createContext, useState } from 'react';
+
+export const MemeContext = createContext({
+ memes: null,
+ setMemes: () => null
+});
+
+export const MemeProvider = ({ children }) => {
```

Comment:
Interesting - I haven't personally run into creating a MemeProvider component this way!